4.4/4.4.5/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 4.4.5
+Создать объект со следующей логикой:
+Появляется алерт с сообщением:
+"Космос получил информацию о вас. Вы - человек [приятно; пузыристо; утончённо; извилисто]-
+[глазастый, ушастый, волосастый, щекастый] [горбунёк, снусмумрик, супермэн, кракен]"
+Значения из квадратных скобок выбираются случайно.
+<script>
+	const infoObj = {
+		randomNum: null,
+		variableValue: null,
```

Comment:
If to be honest you don't need this `variableValue` property for your logic, you can do without it easily